test/extensions/tracers/zipkin/zipkin_tracer_impl_test.cc

```diff
@@ -181,6 +181,7 @@ TEST_F(ZipkinDriverTest, InitializeDriver) {
     const std::string yaml_string = R"EOF(
     collector_cluster: fake_cluster
     collector_endpoint: /api/v1/spans
```

Comment:
Note to reviewer: 
I guess I could update this (and below) to "/api/v2/spans".
But I see in the other places that we are using `/api/v1/spans` with `HTTP_JSON` (e.g. test/extensions/filters/network/http_connection_manager/config_test.cc) so i just keep this unchanged.

Please let me know WDYT.

Reply:
Please just update them to v2 so as to avoid any possible confusion from folks reading the docs.

Reply:
Sure. Done!